bundles/org.openhab.binding.renault/src/main/resources/OH-INF/i18n/renault_fr.properties

```diff
@@ -55,6 +55,9 @@ thing-type.config.renault.car.vin.description = Numéro d'identification du véh
 # channel types
 
 channel-type.renault.batteryavailableenergy.label = Énergie Disponible Batterie
+channel-type.renault.batterystatusupdated.label = État de la batterie mis à jour
```

Comment:
Please remove translation changes from the PR and provide them in Crowdin after the PR has been merged. See https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations